include/seqan3/utility/simd/all.hpp

```diff
@@ -0,0 +1,47 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Marcel Ehrhardt <marcel.ehrhardt AT fu-berlin.de>
+ * \brief Meta-header for the simd module.
+ */
+
+#pragma once
+
+//!\cond DEV
+
+/*!\defgroup simd Simd
+ * \brief The simd module contains a unified interface to provide simd types and functions used in seqan3.
+ * \see https://en.wikipedia.org/wiki/SIMD
+ * \ingroup core
```

Comment:
Thanks to @simonsasse who just corrected me in my PR, I checked yours again. 
This should be in group utility now!
```suggestion
 * \ingroup utility
```

check all other files :see_no_evil:

Reply:
I only have one since the documentation is within the simd module